app/service/document.py

```diff
@@ -124,6 +126,18 @@ def update(
     entity_org_id = get_org_from_id(db, import_id)
     app_user.raise_if_unauthorised_to_make_changes(user, entity_org_id, import_id)
 
+    family = family_service.get(doc.family_import_id)
+    if family is None:
+        raise ValidationError(f"Could not find family for {doc.family_import_id}")
+
+    # Validate metadata.
+    metadata_service.validate_metadata(
+        db,
+        family.corpus_import_id,
+        document.metadata,
+        EntitySpecificTaxonomyKeys.DOCUMENT.value,
```

Comment:
Oh no - was the enum not based on str (see SortOrder)

Reply:
Yes it is - The function `get_entity_specific_taxonomy` takes a str though not the enum, so I've had to do this until I can update  it in the db_client as a driveby

Reply:
thanks